modules/combined/examples/xfem/tests

```diff
@@ -3,10 +3,12 @@
     type = RunApp
     input = 'xfem_mechanics_prescribed_growth.i'
     check_input = True
+    unique_id = true
```

Comment:
What is this all about?

Reply:
I don't have libMesh compiled with --enable-unique-id and this test fails. This just fixes it so it is skipped, like many of the other XFEM. Just something I noticed.

Reply:
Ok